CHARTER.md

```diff
@@ -5,15 +5,85 @@
+develop the file format for describing public software, create awareness and
+educate the world about the benefits of using open source software in
+governments. The group will maintain the file format and keep it aligned with
+the needs of the governments and the agencies. |
+| **3. Purpose and Scope**  | 
+Public software is becoming more and more relevant each day, however its
+discoverability is crucial in order to effectively reuse it also in contexts
+different from the ones where it was originated. 
+As such, creating catalogues where public software is indexed and easily
+searchable is key to stimulate its reuse. However, catalogues need to be
+populated in an automated way to avoid scalability issues and to quickly grow.
+The publiccode.yml format goes exactly in this direction. In
+fact, it is a file that has to be stored in the root of the repository
+of the open source software in question containing the metadata
+definition of such software. In this way, it is quite trivial for automated
+algorithms, such as a crawler, to look for the file, extract relevant
+information from it, create a HTML page dedicated to that software and then insert
+such page in the catalogue.
 
-For more information, please see:
-- [[Open Projects Guide]](https://github.com/oasis-open-projects/documentation/tree/master/guides)
-- [[Getting Started with Open Projects]](https://github.com/oasis-open-projects/documentation/blob/master/guides/getting-started-guide.md)
-- [[Open Project Rules]](https://www.oasis-open.org/policies-guidelines/open-projects-process)
-- [[Open Projects Website]](http://oasis-open-projects.org)
+<br/><br/>The following is an initial list of activities and materials
+which may be developed by the publiccode.yml Open Project working group:<br/><br/>
+<ol>
+    <li>**Standard Development
+        <ol>
+            <li>Definition of the main international keys</li>
+            <li>Definition of recognized rules for Country customization</li>
+            <li>Definition and development of working reference materials</li>
+    <li>**Tools**
+        <ol>
+            <li>Definition of a set of working tools</li>
+            <li>Development of a set of tools</li>
+    <li>**Awareness**
+        <ol>
+            <li>Dedicated Website</li>
+            <li>Media Outreach Campaign</li>
+            <li>Publiccode Conference</li>
+            <li>Publiccode Hackathon</li>
+        </ol>
+    </li>
+    <li>**Education**
+        <ol>
+            <li>What is public software and why we need it?</li>
+            <li>Relationship between public code and open source licenses</li>
+            <li>Importance of having a standard for public code</li>
+            <li>Advantages of the standard</li>
+            <li>Public code in governments</li>
+        </ol>
+    </li>
+    <li>**Marketing**
+        <ol>
+            <li>Impact of public code in an open government perspective</li>
+            <li>Economic value of reuse public software</li>
```

Comment:
I am not sure of this either, but it may be of great impact if carried out involving other govs.

Reply:
What about "Societal impact"?
I'm thinking about transparency of code (and thus of PAs in general) and the cultural approach by which public goods should be publicly managed and developed.
Is something we could specify here as an additional point?